funnel/models/project.py

```diff
@@ -268,6 +269,15 @@ def __repr__(self):
             self.title,
         )
 
+    @property
+    def title_inline(self):
+        """Suffix a colon if the title does not end in ASCII sentence punctuation"""
+        title = self.title.strip()
+        if title and self.tagline.strip():
```

Comment:
Both instances of `.strip()` are only required if we're not validating title and tagline properly to strip whitespace during data entry in the form. That needs to be checked, and if done correctly, these cases of `.strip()` should be removed.

Reply:
Will also require checking the database for trailing whitespace as there may be old data predating the form validation.

Reply:
@qurbat you should investigate this.